crates/ra_ide/src/display/short_label.rs

```diff
@@ -61,7 +61,15 @@ impl ShortLabel for ast::TypeAlias {
 
 impl ShortLabel for ast::Const {
     fn short_label(&self) -> Option<String> {
-        short_label_from_ty(self, self.ty(), "const ")
+        match short_label_from_ty(self, self.ty(), "const ") {
+            Some(buf) => {
+                let mut new_buf = buf;
+                let body = self.body().unwrap();
```

Comment:
This unwrap can fail if I'm not mistaken, in which case the value should just be left out (no panic)